### PR TITLE
turn on online only and distance learning

### PR DESCRIPTION
**Pull request by rianfowler**
## Testing done
verified locally 

## Screenshots
screen shot 2018-11-16 at 09 29 35
screen shot 2018-11-16 at 09 28 42
screen shot 2018-11-16 at 09 28 39
screen shot 2018-11-16 at 09 26 32
screen shot 2018-11-16 at 09 26 27

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/gi/components/profile/Programs.jsx [hunk 1]
```diff
@@ -57,6 +57,18 @@ export class Programs extends React.Component {
         text: 'Priority Enrollment',
         link: false,
       },
+
+      onlineOnly: {
```
Comment (rianfowler) [inline src/applications/gi/components/profile/Programs.jsx]: these properties are tied to the api response. e.g. onlineOnly maps to online_only

**Comment by rianfowler** [comment]
delaying until we have content for the modal

**Review by erikphansen — APPROVED** [review]

**Comment by rianfowler** [comment]
@peggygannon I added some screenshots @peggygannon

**Review by cehsu — APPROVED** [review]

**Review by peggygannon — CHANGES_REQUESTED** [review]
@rianfowler Hi, Rian, seeing this laid out in a modal, I think we should delete these two lines below.  I think we want to focus on the housing allowance in the modal copy. 

Please delete: 
"With online learning, students have access to a teacher and student resources, such as recorded lessons or online forums. With distance learning, the student and teacher are in separate locations and there is little interaction with fellow students."

**Comment by rianfowler** [comment]
@peggygannon I've updated the copy and screen shots in the description.

**Comment by peggygannon** [comment]
@rianfowler Hi.. Can I see the updated screenshot, so i can check/sign off? Thanks!

**Comment by rianfowler** [comment]
@peggygannon 
screen shot 2018-11-16 at 13 12 35

screen shot 2018-11-16 at 13 10 59

**Review by peggygannon — APPROVED** [review]

**Pull request merged**